buildkite/src/Jobs/Release/MinaArtifactMainnetBullseye.dhall

```diff
@@ -21,13 +21,14 @@ in  Pipeline.build
             , Artifacts.Type.Rosetta
             , Artifacts.Type.ZkappTestTransaction
             ]
-          , networks = [ Network.Type.Devnet, Network.Type.Mainnet ]
```

Comment:
@dkijania is this change a good one? It seems not to be required, but I found no use of `devnet` being here among the list of networks. What kind of effect removing it would have on docker images?

Reply:
Alas, it is not so easy. Originally we divided mina builds into two:

- berkeley 
- devnet and mainnet

Unfortunately we need to split devnet and mainnet builds into two separate as they should use different dune profiles. Current code in master does not differ devnet and mainnet and mina binaries are built only once for both networks with profile=devnet. In order to make it right we need to split release jobs into mainnet and devnet. each jobs will have different profile. It is done on develop branch (as it already had ported code for berkeley removal which we reverted from compatible, thus master is not updated). IMHO it's not a big change to make. i can follow up on this pr and cherry pick required commits

Reply:
Please ignore my above comment. We will still build devnet profile under the berkeley name (we didn't remove it from CI ) so as a result we will still build two network (devnet a.k.a berkeley with profile devnet and mainnet with profile mainnet)

I fixed dhall issues in my pr https://github.com/MinaProtocol/mina/pull/16292